scripts/build_header.sh

```diff
@@ -1,9 +1,8 @@
 #! /bin/sh
-
 cargo install --version 0.20.0 cbindgen
 
-rm -f ./scripts/sv2.h
-touch ./scripts/sv2.h
+rm -f ./sv2.h
+touch ./sv2.h
```

Comment:
Why is this change needed?

Reply:
We're calling this script from `sv2-header-check.sh`, which changes the PWD to the scripts directory before invoking `build_header.sh`. The buildr_header shell script point to the scripts folder, but since it was already inside the scripts directory, the removal and creation commands couldn't locate the correct path. As a result, it couldn't create the `sv2.h` file at the desired location and instead appended the `cbindgen` output to the older version of `sv2.h`. This caused the hash test to fail.